packages/runtime-core/src/components/Teleport.ts

```diff
@@ -130,7 +130,7 @@ export const TeleportImpl = {
       const currentAnchor = wasDisabled ? mainAnchor : targetAnchor
 
       if (n2.dynamicChildren) {
-        // fast path when the teleport happens to be a block root
+        // fast patch when the teleport happens to be a block root
```

Comment:
"path" is correct here
```suggestion
        // fast path when the teleport happens to be a block root
```